--- 56028828\data.inc	2021-12-23 12:26:21.263433600 -0800
+++ 56028832\data.inc	2022-01-08 19:44:08.887395000 -0800
@@ -5,15 +5,15 @@
 unk_8002:   fcb $56
             fcb $02
             fcb $88
-byte_8005:  fcb $28
+byte_8005:  fcb $32
 byte_8006:  fcb $A1
 unk_8007:   fcb $56
             fcb $02
             fcb $88
-            fcb $28
+            fcb $32
 byte_800B:  fcb $A1
-            fcb $8D
-unk_800D:   fcb $4E
+            fcb $E7
+unk_800D:   fcb $4A
             fcb $18
 byte_800F:  fcb $15
             fcb $88
@@ -33,7 +33,7 @@
 byte_8015:  fcb %10000000+TransType
 ;------------------------------------------------------------------------------
 TL_ConfigurationFlags2:
-byte_8016:  fcb $22
+byte_8016:  fcb $42
 
 TL_Amount_MINTHR_IsIncreasedAtKeyOnOnly:
 byte_8017:  fcb $03
@@ -89,7 +89,7 @@
 word_807A:  fdb $53B5
 word_807C:  fdb $7530
 byte_807E:  fcb $00
-byte_807F:  fcb $14
+byte_807F:  fcb $04
 byte_8080:  fcb $02
 byte_8081:  fcb $04
 byte_8082:  fcb $05
@@ -182,7 +182,7 @@
 byte_827B:  fcb $1C
 
 TL_TL_827C:
-unk_827C:   fcb $00, $00, $00, $00, $00, $00, $00, $00, $00
+unk_827C:   fcb $14, $14, $0A, $00, $00, $00, $00, $00, $00
 
 TL_TL_8285:
 unk_8285:   fcb $FF, $FF, $FF, $FF, $FF, $FF, $FF, $FF, $FF
@@ -203,11 +203,11 @@
             fcb $00
             fcb $3E, $3E, $3E, $3E, $3E, $3E, $42, $52, $6C
             fcb $50, $50, $4A, $4A, $4A, $50, $50, $60, $80
-            fcb $50, $50, $4A, $50, $57, $58, $5C, $69, $8D
-            fcb $4A, $60, $5B, $64, $6A, $68, $68, $6B, $86
-            fcb $7A, $7A, $8E, $7D, $73, $70, $70, $6D, $80
-            fcb $8A, $8A, $95, $7E, $7B, $74, $72, $76, $7A
-            fcb $90, $9A, $9D, $8C, $7F, $7A, $77, $77, $7A
+            fcb $50, $50, $46, $46, $54, $58, $5C, $69, $8D
+            fcb $4A, $60, $46, $46, $5A, $68, $68, $6B, $86
+            fcb $6A, $6A, $60, $60, $60, $70, $70, $6D, $80
+            fcb $80, $80, $70, $70, $7B, $74, $72, $76, $7A
+            fcb $80, $80, $80, $80, $7F, $7A, $77, $77, $7A
             fcb $A0, $9A, $9D, $88, $7D, $7A, $79, $7A, $79
             fcb $A0, $A0, $9D, $91, $80, $7B, $7A, $7C, $78
             fcb $A0, $B0, $9D, $86, $7A, $7D, $7A, $7C, $7E
@@ -243,27 +243,27 @@
             fcb $4A, $4A, $4A, $55, $5F, $5E, $5F, $5F, $5F
 
 TL_TL_83EB:
-unk_83EB:   fcb $D0, $A4, $74, $13, $0C, $0A, $04, $04, $04
+unk_83EB:   fcb $B0, $64, $34, $13, $0C, $08, $04, $04, $04
 
 TL_TL_83F4:
 unk_83F4:   fcb $14, $14, $14, $14, $10, $0A, $03, $03, $03
 
 TL_TL_83FD:
-unk_83FD:   fcb $CA, $88, $58, $2A, $23, $0F, $05, $00, $00
+unk_83FD:   fcb $CA, $7A, $34, $2A, $23, $0F, $05, $03, $00
 
 TL_TL_8406:
-unk_8406:   fcb $50, $48, $34, $1E, $10, $08, $00, $00, $00
+unk_8406:   fcb $50, $30, $18, $18, $14, $0C, $00, $00, $00
 
 TL_TL_840f:
-unk_840F:   fcb $00, $00, $00, $00, $00, $40, $80, $80, $80
+unk_840F:   fcb $00, $00, $00, $00, $40, $60, $60, $80, $40
 
 TL_TL_8418:
 unk_8418:   fcb $00, $00, $00, $00, $00, $00, $00, $00, $00
 
 TL_TL_8421:
-unk_8421:   fcb $40, $40, $30, $2E, $28, $20, $1A, $08, $08
+unk_8421:   fcb $40, $40, $28, $28, $28, $20, $10, $04, $04
 
-byte_842A:  fcb $00
+byte_842A:  fcb $10
 
 unk_842B:
             fcb $02
@@ -290,7 +290,7 @@
 
 unk_846F:
             fcb $04
-            fcb $0C, $00, $00, $AB
+            fcb $1C, $00, $01, $33
             fcb $30, $18, $00, $D5
             fcb $60, $40, $02, $00
             fcb $80, $80
@@ -319,9 +319,9 @@
 
 unk_84AC:   fcb $20, $20, $20, $18, $18, $18, $18, $18, $18
 
-unk_84B5:   fcb $CF, $A0, $7D, $32, $2A, $28, $1C, $1C, $16
+unk_84B5:   fcb $A0, $80, $5C, $35, $20, $16, $16, $16, $16
 
-unk_84BE:   fcb $60, $50, $40, $32, $2C, $28, $10, $10, $10
+unk_84BE:   fcb $60, $30, $28, $28, $20, $20, $10, $10, $10
 
 unk_84C7:   fcb $20, $20, $20, $20, $20, $40, $40, $40, $40
 
@@ -342,32 +342,32 @@
             fcb $40
 byte_84E1:  fcb $F0
 
-unk_84E2:   fcb $28, $28, $24, $1D, $1B, $18, $18, $18, $18
+unk_84E2:   fcb $28, $28, $24, $1D, $1B, $19, $19, $18, $18
 
 byte_84EB:  fcb $10
 word_84EC:  fdb $024F
 
 TL_RPM3DInputModifierFromRPM2:
 unk_84EE:
-            fcb $04
+            fcb $05
             fcb $13, $00, $0A, $AB
+            fcb $19, $40, $0A, $AB
             fcb $1F, $80, $02, $11
             fcb $3E, $C0, $01, $57
             fcb $6D, $FF
 
-
- DCB.B $10, $FF ; 16 bytes of 0xFF
+ DCB.B $0C, $FF ; 12 bytes of 0xFF
 
 TL_O2LoLimitForInMiddleDetermination:
 unk_850D:
             fcb $05
             fcb $09
             fcb $00
-            fcb $1C, $1C, $1C, $1C, $1C, $1C, $1C, $1C, $1C
-            fcb $1C, $1C, $1C, $1C, $1C, $1C, $1C, $1C, $1C
-            fcb $1C, $1C, $1C, $1C, $1C, $1C, $1C, $1C, $1C
-            fcb $18, $18, $18, $18, $18, $18, $18, $18, $18
-            fcb $18, $18, $18, $18, $18, $18, $18, $18, $18
+            fcb $1D, $1D, $1D, $1D, $1D, $1D, $1D, $1D, $1D
+            fcb $1D, $1D, $1D, $1D, $1D, $1D, $1D, $1D, $1D
+            fcb $1D, $1D, $1D, $1D, $1D, $1D, $1D, $1D, $1D
+            fcb $1A, $1A, $1A, $1A, $1A, $1A, $1A, $1A, $1A
+            fcb $1A, $1A, $1A, $1A, $1A, $1A, $1A, $1A, $1A
 
  DCB.B $6C, $FF ; 108 bytes of 0xFF
 
@@ -376,11 +376,11 @@
             fcb $05
             fcb $09
             fcb $00
-            fcb $22, $22, $22, $22, $22, $22, $22, $22, $22
-            fcb $22, $22, $22, $22, $22, $22, $22, $22, $22
-            fcb $22, $22, $22, $22, $22, $22, $22, $22, $22
-            fcb $20, $20, $20, $20, $20, $20, $20, $20, $20
-            fcb $20, $20, $20, $20, $20, $20, $20, $20, $20
+            fcb $21, $21, $21, $21, $21, $21, $21, $21, $21
+            fcb $21, $21, $21, $21, $21, $21, $21, $21, $21
+            fcb $21, $21, $21, $21, $21, $21, $21, $21, $21
+            fcb $1E, $1E, $1E, $1E, $1E, $1E, $1E, $1E, $1E
+            fcb $1E, $1E, $1E, $1E, $1E, $1E, $1E, $1E, $1E
 
  DCB.B $6C, $FF ; 108 bytes of 0xFF
 
@@ -397,7 +397,7 @@
 unk_864F:   fcb $40
 byte_8650:  fcb $0B
 byte_8651:  fcb $05
-byte_8652:  fcb $8C
+byte_8652:  fcb $A0
 unk_8653:   fcb $C0
 byte_8654:  fcb $0B
 byte_8655:  fcb $15
@@ -459,28 +459,30 @@
 byte_8694:  fcb $06
 word_8695:  fdb $B220
 word_8697:  fdb $0190
-byte_8699:  fcb $80
+byte_8699:  fcb $A0
 byte_869A:  fcb $01
-word_869B:  fdb $00B6
+word_869B:  fdb $0040
 byte_869D:  fcb $0F
 byte_869E:  fcb $1C
 byte_869F:  fcb $1C
 
 TL_SparkAdvanceAddedWhenRpmBelowControl_FromDeltaRpm:
 unk_86A0:
-            fcb $02
-            fcb $05, $00, $01, $6E
+            fcb $03
+            fcb $00, $00, $00, $CD
+            fcb $05, $04, $00, $DB
             fcb $0C, $0A
 
- DCB.B $04, $FF ; 4 bytes of 0xFF
+
 
 TL_SparkAdvanceRetardedWhenRpmAboveControl_FromDeltaRpm:
 unk_86AB:
-            fcb $02
-            fcb $05, $00, $FF, $9A
-            fcb $0F, $FC
+            fcb $03
+            fcb $00, $00, $FF, $9A
+            fcb $05, $FE, $FF, $43
+            fcb $18, $F0
+
 
- DCB.B $04, $FF ; 4 bytes of 0xFF
 
 TL_MinVoltageForAis:
 byte_86B6:  fcb $71
@@ -491,11 +493,11 @@
 TL_AisStartupPosition_FromTemp:
 unk_86C2:
             fcb $05
-            fcb $58, $E0, $FE, $66
-            fcb $80, $A0, $FD, $CD
-            fcb $A8, $48, $FF, $9A
-            fcb $D0, $38, $00, $CD
-            fcb $F8, $58
+            fcb $58, $D8, $FE, $66
+            fcb $80, $98, $FD, $CD
+            fcb $A8, $40, $FF, $9A
+            fcb $D0, $30, $00, $CD
+            fcb $F8, $50
 
 unk_86D5:
             fcb $03
@@ -503,19 +505,19 @@
             fcb $C8, $10, $FF, $9A
             fcb $F0, $00
 
-byte_86E0:  fcb $11
-byte_86E1:  fcb $A0
-byte_86E2:  fcb $20
+byte_86E0:  fcb $14
+byte_86E1:  fcb $94
+byte_86E2:  fcb $2B
 byte_86E3:  fcb $D0
 byte_86E4:  fcb $EE
 
 TL_ACOffIdleDelta_FromSpeed:
 unk_86E5:
             fcb $04
-            fcb $14, $14, $FF, $C0
-            fcb $3C, $0A, $FF, $F3
-            fcb $64, $08, $FF, $EF
-            fcb $82, $06
+            fcb $14, $14, $00, $00
+            fcb $3C, $14, $00, $00
+            fcb $50, $14, $00, $53
+            fcb $78, $21
 
 TL_AISAddedFromTimeEngRunning:
 byte_86F4:  fcb $00
@@ -525,8 +527,8 @@
             fcb $04
             fcb $68, $50, $FF, $55
             fcb $80, $40, $FF, $33
-            fcb $A8, $20, $FF, $33
-            fcb $D0, $00
+            fcb $A8, $20, $FF, $46
+            fcb $D4, $00
 
 unk_8704:
             fcb $03
@@ -559,19 +561,19 @@
 TL_IdleSpeedParkNeutral_FromTemp:
 unk_8738:
             fcb $05
-            fcb $58, $96, $FF, $86
-            fcb $A8, $70, $FF, $86
-            fcb $D0, $5D, $00, $00
-            fcb $E5, $5D, $04, $1C
-            fcb $EE, $82
+            fcb $58, $96, $FF, $8D
+            fcb $A8, $72, $FF, $D1
+            fcb $D4, $6A, $00, $00
+            fcb $E5, $6A, $02, $E4
+            fcb $EE, $84
 
 TL_IdleSpeedDrive_FromTemp:
 unk_874B:
             fcb $05
-            fcb $58, $96, $FF, $86
-            fcb $A8, $70, $FF, $86
-            fcb $D0, $5D, $00, $00
-            fcb $E5, $5D, $04, $1C
+            fcb $58, $96, $FF, $8D
+            fcb $A8, $72, $FF, $C0
+            fcb $D4, $67, $00, $00
+            fcb $E5, $67, $03, $00
             fcb $EE, $82
 
 TL_DecelAisBaseAirflowCurveOffset_FromTime2:
@@ -581,8 +583,8 @@
             fcb $00, $00, $00, $00
             fcb $00, $00
 
-byte_8769:  fcb $67
-byte_876A:  fcb $5D
+byte_8769:  fcb $6A
+byte_876A:  fcb $6A
 
 TL_ForwardAisSteps_NoSlope:
 unk_876B:
@@ -590,21 +592,21 @@
             fcb $32, $01, $00, $00
             fcb $4B, $02, $00, $00
             fcb $5A, $03, $00, $00
-            fcb $78, $05, $00, $00
-            fcb $96, $08
+            fcb $78, $03, $00, $00
+            fcb $96, $04
 
 TL_ReverseAisSteps_NoSlope:
 unk_877E:
             fcb $05
             fcb $40, $FF, $00, $00
-            fcb $A0, $FF, $00, $00
-            fcb $C8, $FF, $00, $00
-            fcb $E6, $FF, $00, $00
-            fcb $FF, $FF
-
-byte_8791:  fcb $11
-byte_8792:  fcb $11
-byte_8793:  fcb $EE
+            fcb $A0, $FE, $00, $00
+            fcb $C8, $FB, $00, $00
+            fcb $E6, $FA, $00, $00
+            fcb $FF, $F8
+
+byte_8791:  fcb $0A
+byte_8792:  fcb $0A
+byte_8793:  fcb $EB
 
 unk_8794:   fcb $1A, $1A, $1A, $1A, $1A, $1A, $16, $14, $14
 
@@ -625,7 +627,7 @@
 byte_87AB:  fcb $7F
 
 TL_PositionAisMotorIsAtIdle:
-byte_87AC:  fcb $10
+byte_87AC:  fcb $18
 
 TL_AISAddedFromTimeEngRunning2:
 byte_87AD:  fcb $08
@@ -633,16 +635,16 @@
 TL_DeltaRpmChangeToUse_DRPNK_or_PNDRK:
 unk_87AE:
             fcb $03
-            fcb $58, $F0, $00, $00
-            fcb $A8, $F0, $00, $00
-            fcb $F8, $F0
+            fcb $58, $E0, $00, $00
+            fcb $90, $E0, $00, $40
+            fcb $D0, $F0
 
 TL_DeltaRpmChangeToUse_DRPNK_or_PNDRK2
 unk_87B9:
             fcb $03
-            fcb $58, $10, $00, $00
-            fcb $A8, $10, $00, $00
-            fcb $F8, $10
+            fcb $58, $30, $FF, $B7
+            fcb $90, $20, $FF, $C0
+            fcb $D0, $10
 
 TL_AISDecel_FromBaro:
 unk_87C4:
@@ -697,34 +699,34 @@
 TL_DecelAisOpenStepsFromNeg_FromDeltaMap:
 unk_881A:
             fcb $04
-            fcb $08, $00, $00, $80
-            fcb $0C, $02, $00, $40
-            fcb $10, $03, $00, $40
-            fcb $18, $05
+            fcb $0C, $00, $01, $00
+            fcb $10, $04, $00, $40
+            fcb $18, $06, $00, $40
+            fcb $20, $08
 
 TL_DecelAisOpenStepsFromPos_FromDeltaMap:
 unk_8829:
             fcb $04
             fcb $08, $00, $FF, $C0
             fcb $0C, $FF, $FF, $C0
-            fcb $10, $FE, $FF, $E0
-            fcb $18, $FD
+            fcb $10, $FE, $00, $00
+            fcb $18, $FE
 
 TL_TimeBetweenAisPulsesForDecelAis:
 byte_8838:  fcb $3F
 
 TL_EngineTempToStartCalibratingDacal:
-byte_8839:  fcb $D6
+byte_8839:  fcb $D4
 
 TL_DeltaRpmFromIdleSpeedForDacalUpdate:
 byte_883A:  fcb $02
 
 TL_ACFactorDeltaFromBase:
-byte_883B:  fcb $12
-byte_883C:  fcb $1F
+byte_883B:  fcb $06
+byte_883C:  fcb $38
 
 TL_Base_nominalCal:
-byte_883D:  fcb $00
+byte_883D:  fcb $03
 
 TL_DacalInhibitTimeAfterStartup:
 byte_883E:  fcb $0B
@@ -740,7 +742,7 @@
 byte_8847:  fcb $FF
 word_8848:  fdb $FF00
 word_884A:  fdb $FF00
-byte_884C:  fcb $04
+byte_884C:  fcb $00
 byte_884D:  fcb $B6
 byte_884E:  fcb $07
 byte_884F:  fcb $10
@@ -771,43 +773,43 @@
 word_8861:  fdb $0100
 byte_8863:  fcb $25
 byte_8864:  fcb $C0
-byte_8865:  fcb $92
-byte_8866:  fcb $00
-byte_8867:  fcb $20
-byte_8868:  fcb $32
+byte_8865:  fcb $58
+byte_8866:  fcb $01
+byte_8867:  fcb $00
+byte_8868:  fcb $00
 byte_8869:  fcb $00
-byte_886A:  fcb $06
-byte_886B:  fcb $40
+byte_886A:  fcb $00
+byte_886B:  fcb $00
 
 unk_886C:
             fcb $06
-            fcb $1C, $24, $00, $E7
-            fcb $3B, $40, $05, $C0
-            fcb $3F, $57, $06, $AB
-            fcb $42, $6B, $00, $00
-            fcb $54, $6B, $00, $00
-            fcb $54, $A8
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00
 
 unk_8883:
             fcb $04
-            fcb $1F, $0F, $00, $F7
-            fcb $74, $61, $00, $00
-            fcb $8A, $61, $00, $00
-            fcb $8A, $A8
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00
 
  DCB.B $08, $FF ; 8 bytes of 0xFF
 
 unk_889A:
             fcb $04
-            fcb $1F, $0F, $00, $F7
-            fcb $74, $61, $00, $00
-            fcb $8A, $61, $00, $00
-            fcb $8A, $A8
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00, $00, $00
+            fcb $00, $00
 
  DCB.B $08, $FF ; 8 bytes of 0xFF
 
 TL_PTUMinLockTemperature:
-byte_88B1:  fcb $B8
+byte_88B1:  fcb $58
 
 TL_PTUCounterTimerIntervalForLoop:
 byte_88B2:  fcb $07
@@ -816,69 +818,71 @@
 byte_88B3:  fcb $7C
 
 TL_PTUMAPThresholdForLock3:
-byte_88B4:  fcb $14
+byte_88B4:  fcb $C0
 
 TL_PTUDeltaTPSForUnlock:
-byte_88B5:  fcb $66
+byte_88B5:  fcb $40
 
 TL_PTUMAPThresholdForLock1:
-unk_88B6:   fcb $81, $18
+unk_88B6:   fcb $E3, $E3
 
 TL_PTUCounterHiSpeed:
-byte_88B8:  fcb $20
+byte_88B8:  fcb $02
 
 TL_PTUCounterLoSpeed:
-byte_88B9:  fcb $11
+byte_88B9:  fcb $02
 
 byte_88BA:  fcb $00
 
 TL_UpperLimitForPTULock:
 unk_88BB:
             fcb $04
-            fcb $64, $0F, $00, $00
-            fcb $64, $2E, $00, $AB
-            fcb $82, $42, $00, $00
-            fcb $82, $A8
+            fcb $4C, $1E, $06, $9A
+            fcb $56, $60, $00, $7A
+            fcb $82, $75, $04, $4D
+            fcb $8C, $A0
 
 TL_LowerLimitForPTULock:
-            fcb $02
-            fcb $64, $0F, $00, $00
-            fcb $96, $0F
+            fcb $03
+            fcb $4C, $1E, $00, $00
+            fcb $4C, $08, $00, $00
+            fcb $96, $08
 
- DCB.B $08, $FF ; 8 bytes of 0xFF
+ DCB.B $04, $FF ; 8 bytes of 0xFF
 
 TL_UpperLimitForPTUUnlock:
 unk_88D9:
             fcb $04
-            fcb $5E, $05, $00, $00
-            fcb $5E, $33, $00, $AB
-            fcb $7C, $47, $00, $00
-            fcb $7C, $A8
+            fcb $46, $1E, $06, $9A
+            fcb $50, $60, $00, $7A
+            fcb $7C, $75, $04, $4D
+            fcb $86, $A0
 
 TL_LowerLimitForPTUUnlock:
-            fcb $02
-            fcb $5E, $05, $00, $00
-            fcb $96, $05
+            fcb $03
+            fcb $46, $1E, $00, $00
+            fcb $46, $03, $00, $00
+            fcb $96, $03
 
- DCB.B $08, $FF ; 8 bytes of 0xFF
+ DCB.B $04, $FF ; 8 bytes of 0xFF
 
-byte_88F7:  fcb $50
+byte_88F7:  fcb $00
 byte_88F8:
-TL_TL_88E4: fcb $2D
-byte_88F9:  fcb $28
-byte_88FA:  fcb $0A
-byte_88FB:  fcb $06
-byte_88FC:  fcb $05
-byte_88FD:  fcb $10
-byte_88FE:  fcb $0F
-byte_88FF:  fcb $50
-byte_8900:  fcb $08
-
-unk_8901:   fcb $FF, $FF, $FF, $41, $41, $3F, $3C, $40, $40
-
-byte_890A:  fcb $08
-byte_890B:  fcb $0D
-byte_890C:  fcb $80
+TL_TL_88E4: fcb $00
+byte_88F9:  fcb $00
+byte_88FA:  fcb $00
+byte_88FB:  fcb $00
+byte_88FC:  fcb $00
+byte_88FD:  fcb $00
+byte_88FE:  fcb $00
+byte_88FF:  fcb $00
+byte_8900:  fcb $00
+
+unk_8901:   fcb $00, $00, $00, $00, $00, $00, $00, $00, $00
+
+byte_890A:  fcb $00
+byte_890B:  fcb $00
+byte_890C:  fcb $00
 byte_890D:  fcb $FE
 byte_890E:  fcb $F0
 byte_890F:  fcb $00
@@ -915,31 +919,31 @@
 byte_8928:  fcb $06
 
 TL_UnknownConstForCruiseControl36:
-byte_8929:  fcb $5A
+byte_8929:  fcb $74
 
 TL_CruiseControlSetDeltaControlPointForCableAdaptive:
-byte_892A:  fcb $04
+byte_892A:  fcb $02
 
 TL_CruiseCompareConstant2:
-byte_892B:  fcb $20
+byte_892B:  fcb $04
 
 TL_CruiseMask2:
 byte_892C:  fcb $10
 
 TL_CruiseMask3:
-byte_892D:  fcb $FC
+byte_892D:  fcb $F0
 
 TL_CruiseMask7:
 byte_892E:  fcb $04
 
 TL_CruiseMask4:
-byte_892F:  fcb $08
+byte_892F:  fcb $04
 
 TL_CruiseMask6:
-byte_8930:  fcb $42
+byte_8930:  fcb $28
 
 TL_CruiseMask5:
-byte_8931:  fcb $CE
+byte_8931:  fcb $D8
 
 TL_CruiseCompareConstant1:
 byte_8932:  fcb $0B
@@ -950,10 +954,10 @@
 TL_CruiseControlVentRegularGainCurve_FromDeltaSpeed:
 unk_8934:
             fcb $04
-            fcb $00, $00, $FF, $FB
+            fcb $18, $00, $FF, $F5
             fcb $30, $FF, $FF, $F6
             fcb $80, $FC, $FF, $EC
-            fcb $FF, $F2
+            fcb $FE, $F2
 
 TL_CruiseControlVacRegularGainCurve_FromDeltaSpeed:
 unk_8943:
@@ -1006,15 +1010,15 @@
 TL_CruiseControlSpeedGainFactor_FromSpeed:
 unk_89AE:
             fcb $04
-            fcb $46, $04, $FF, $F3
-            fcb $5A, $03, $FF, $F3
-            fcb $6E, $02, $FF, $E6
-            fcb $82, $00
+            fcb $46, $0C, $00, $00
+            fcb $5A, $0C, $FF, $E6
+            fcb $6E, $0A, $FF, $E6
+            fcb $82, $08
 
 TL_CruiseControlThrottleGainFactor_FromThrottle:
 unk_89BD:
             fcb $04
-            fcb $00, $06, $00, $E0
+            fcb $00, $12, $00, $20
             fcb $10, $14, $00, $90
             fcb $20, $1D, $00, $30
             fcb $30, $20
@@ -1037,18 +1041,18 @@
 TL_UnknownTableForCruiseControl2:
 unk_89E6:
             fcb $02
-            fcb $A0, $03, $00, $18
-            fcb $C0, $06
+            fcb $90, $00, $00, $60
+            fcb $A0, $06
 
  DCB.B $04, $FF ; 4 bytes of 0xFF
 
 TL_UnknownTableForCruiseControl3:
 unk_89F1:
-            fcb $02
-            fcb $A8, $80, $FE, $ED
-            fcb $D0, $55
+            fcb $03
+            fcb $90, $FF, $FA, $B5
+            fcb $A8, $80, $FC, $CD
+            fcb $D0, $00
 
- DCB.B $04, $FF ; 4 bytes of 0xFF
 
 TL_UnknownTableForCruiseControl4:
 unk_89FC:
@@ -1073,20 +1077,20 @@
 byte_8A19:  fcb $02
 byte_8A1A:  fcb $02
 byte_8A1B:  fcb $80
-byte_8A1C:  fcb $00
-byte_8A1D:  fcb $00
+byte_8A1C:  fcb $28
+byte_8A1D:  fcb $40
 
-unk_8A1E:   fcb $00, $00, $00, $00, $00, $00, $00, $00, $00
+unk_8A1E:   fcb $14, $14, $14, $14, $14, $14, $14, $14, $14
 
-byte_8A27:  fcb $00
-byte_8A28:  fcb $00
+byte_8A27:  fcb $0C
+byte_8A28:  fcb $C0
 
-unk_8A29:   fcb $00, $00, $00, $00, $00, $00, $00, $00, $00
+unk_8A29:   fcb $50, $38, $20, $18, $10, $08, $04, $00, $00
 
 TL_ErrorBitsMask1:
 unk_8A32:   fcb $26
             fcb $00, $00, $00, $00, $00
-byte_8A38:  fcb $04
+byte_8A38:  fcb $24
 byte_8A39:  fcb $11
 
 TL_PowerLossLightMask:
